Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
```

Comment:
I think we should update the version to reflect there was a parser for schema V1, even though so much changed.

Reply:
updated to 0.2